codespell_lib/data/dictionary.txt

```diff
@@ -27894,6 +27910,7 @@ viwers->viewers
 vlarge->large
 vlaue->value
 vlaues->values
+vlone->clone
```

Comment:
Can you add the clones and cloned versions of this typo too.

Reply:
Done: 79af33c